lib/accessibility-element-view.js

```diff
@@ -19,6 +19,15 @@ class AccessibilityElementView extends SelectableView {
 
   render () {
     this.selectorPath.textContent = this.path
+
+    // Add a click-handler that will select the element
+    this.selectorPath.onclick = (evt) => {
+      evt.preventDefault()
+      chrome.devtools.inspectedWindow.eval(
+        "inspect($$(\"" + this.path.replace('"', '\\"') + "\")[0])",
+        function(result, isException) { }
```

Comment:
There is a internal helper that is promise-based instead of callback-based you might want to use here.

https://github.com/electron/devtron/blob/master/lib/eval.js#L4